Add configuration and classes for the Eitr Refinery

shawnwallace opened the pull request:
Uses the same mechanics as the other "smelters", except with different materials.

shawnwallace committed:
Add configuration and classes for the Eitr Refinery

Uses the same mechanics as the other "smelters", except with different materials.

JcBernack commented on ValheimPlus/Configurations/Sections/EitrRefineryConfiguration.cs:
```diff
@@ -0,0 +1,13 @@
+        public int maximumSap { get; internal set; } = 20;
+        public int maximumSoftTissue { get; internal set; } = 20;
+        public float productionSpeed { get; internal set; } = 40;
+        public bool autoDeposit { get; internal set; } = true;
+        public bool autoFuel { get; internal set; } = true;
```
Should probably default to `false` for consistency with the other "semelters".

JcBernack reviewed (dismissed):
Nice, was looking for this last weekend  :+1:

shawnwallace committed:
Nits for the config defaults

Updating the Eitr Refinery defaults to match others.  Also updating the Spinning Wheel.

Updated the .csproj to build the new file as well.

MueR commented:
@JcBernack any chance this could get merged? Really hoping to get this soon.

JcBernack commented:
> **[MueR](/MueR) ** commented

Hehe, me too, but I am not a maintainer here.

healiha commented:
Approved but only @nxPublic can merge on the project.

Thanks for fixing some old issues too!
